addon/pods/components/frost-tabs/template.hbs

```diff
@@ -1,6 +1,12 @@
-<div class="tab-nav">
-  {{#each tabs as |tab|}}
-    {{frost-button class=(concat 'tab' (if (eq tab.id selection) ' active')) design='tab' disabled=tab.disabled onClick=(action onChange tab.id) text=tab.alias}}
+<div class="tab-nav" data-test={{hook (concat hook '-tabs') }}>
+  {{#each tabs as |tab index|}}
+    <div data-test={{hook (concat hook '-tabs-' index) }}>
+      {{component tab hook=hook targetOutlet=targetOutlet selectedTab=selectedTab onChange=onChange}}
```

Comment:
Since we know the tabs aren't generic components (unless we intend to substitute the tab itself with another component?) - can we just make this

`{{frost-tab ...`?